app/models/stats.rb

```diff
@@ -4,25 +4,34 @@ class Stats
   attr_reader :new_reg, :assisted_pc, :email_renewals
 
   def initialize
-    @new_reg = calculate_new_reg
+    @reg_from_last_week = retrieve_reg_from_last_week
+    @new_reg = count_new_reg
```

Comment:
You don't need these if you implement this pattern https://github.com/DEFRA/waste-exemptions-front-office/pull/230/files#r324585720 :)